src/webhooks/registry.ts

```diff
@@ -140,6 +166,8 @@ function buildQuery(
 
```

Comment:
I was going to say that we should just update the callers of this method to provide an object of the keys/values and validate it on our side rather than exposing the pubsub uri "format", but then I noticed that this function is exported and who knows how it's used.

In the future, we should probably split this function up into multiple, one per delivery method. But for now, this should be fine, and come to think of it, we don't necessarily need to have the `pubsub://` in front, just `<project>:<topic>` should work fine (something to consider).

Reply:
Agreed.

Too much depends on that `path` argument right now. The most significant thing is that there's a local registry of webhooks, with each webhook having an associated handler function. I feel like it doesn't make sense to store EventBridge or PubSub webhooks in this registry (those webhooks are delivered to EventBridge/PubSub and handled there), so in the future, I'd suggest making an entirely separate function for registering non-HTTP webhooks, and deprecate this behavior in the current `register` function (so it will be used only for HTTP webhooks). However I think we need more context on how exactly this function is being used by partners.